src/ergw_aaa_radius.erl

```diff
@@ -359,6 +360,9 @@ vendor_ituma('WLAN-Station-RSSI ', #{'WLAN-Station-RSSI' := Value}, Attrs) ->
     [{?'IM_UE_Sta_RSSI', Value}|Attrs];
 vendor_ituma('WLAN-AP-RSSI', #{'WLAN-AP-RSSI' := Value}, Attrs) ->
     [{?'IM_UE_Wwan_RSSI', Value}|Attrs];
+vendor_ituma('LI-Location', #{'IM-LI-Location' := Location}, Attrs) ->
+    ?LOG(debug, "Adding LI-Location (decimal string): ~s", [Location]),
```

Comment:
Are you sure that this should be `~s`? What is the type of `IM-LI-Location`and default value?

Reply:
The intended type is a binary afaict, which will be handled by ~s, but it's true that ~p as a default is probably better.

Reply:
In any case this would probably be considered excessive logging by Andreas so I'll remove this message instead.